### PR TITLE
HPCC-16191 Add missing alerts in ECL Watch

### PR DESCRIPTION
**Pull request by miguelvazq**
Fixes HPCC-16191

Signed-off by: Miguel Vazquez miguel.vazquez@lexisnexis.com

### CODE REVIEW

**Comment by miguelvazq** [comment]
@GordonSmith Please review. FYI @wangkx

File: esp/src/eclwatch/InfoGridWidget.js [hunk 1]
```diff
@@ -398,6 +402,12 @@ define([
                                 data.push(item);
                             }
                             break;
+                        case "Alert":
+                        this._counts.alert++;
```
Comment (GordonSmith) [inline esp/src/eclwatch/InfoGridWidget.js]: Minor: indentation

File: esp/src/eclwatch/InfoGridWidget.js [hunk 2]
```diff
@@ -427,6 +437,10 @@ define([
                         return -1;
                     } else if (r.Severity === "Warning") {
                         return 1;
+                    } else if (l.Severity === "Alert") {
```
Comment (GordonSmith) [inline esp/src/eclwatch/InfoGridWidget.js]: Given your coloring alerts the same as errors, they should probably get sorted above warnings, but after errors?

File: esp/src/eclwatch/InfoGridWidget.js [hunk 3]
```diff
@@ -316,7 +319,7 @@ define([
                 var item = this.infoGrid.getItem(row.index);
                 if (item) {
                     var severity = this.store.getValue(item, "Severity", null);
-                    if (severity == "Error") {
+                    if (severity == "Error" || severity === "Alert") {
```
Comment (GordonSmith) [inline esp/src/eclwatch/InfoGridWidget.js]: Another Red-ish color might be nice?

**Comment by miguelvazq** [comment]
@GordonSmith Modified.

**Comment by GordonSmith** [comment]
Looks good.

**Pull request merged**